src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -427,6 +460,7 @@ static void Main()
             Assert.Null(model.GetTypeInfo(def).Type);
             Assert.Null(model.GetSymbolInfo(def).Symbol);
             Assert.Null(model.GetDeclaredSymbol(def));
+            Assert.Equal(0, model.GetConstantValue(def).Value);
```

Comment:
I don't think I fully agree with that. The type of the constant value should agree with the *converted type* of `def`.